build.sbt

```diff
@@ -0,0 +1,96 @@
+name := "play-json-ops-root"
+organization in ThisBuild := "me.jeffmay"
```

Comment:
I'm not sure the details of the transfer, but if you are giving ownership to Rally this should probably be Rally